### PR TITLE
Reviewer dashboard

### PR DESCRIPTION
**Pull request by Dsm0**
working mockup of a reviewer creation/editing page from @yihongli

### CODE REVIEW

**Comment by Dsm0** [comment]
dent in #13 #15 #18 #19 and #21

File: mayan/apps/reviewers/models.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from mayan.apps.databases.model_mixins import ExtraDataModelMixin
+from mptt.models import MPTTModel
+
+class Dashboard(ExtraDataModelMixin, MPTTModel):
+    pass
+
+class Candidate(ExtraDataModelMixin, MPTTModel):
```
Comment (MaElisa) [inline mayan/apps/reviewers/models.py:8]: We may want to add more data validation such as regex, but for now this is good. I'll add a task so we can follow up on this.

**Review by MaElisa — APPROVED** [review]
Left comments on files, but it looks good to me!

**Comment by MaElisa** [comment]
Feel free to do the honors of merging :)

File: mayan/apps/reviewers/models.py (hunk 1 above)
Comment (MaElisa) [reply mayan/apps/reviewers/models.py:8]: Created issue #33 to close the loop on this.

**Pull request merged**